misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/testUtils.kt

```diff
@@ -182,6 +181,20 @@ fun <T, R> unifySignatures(
     )
 }
 
+// TODO I cant make this work any way I try, it seems to me like I wont be able to unify like this
+//@JvmName("unifySignatures0Feature")
+//fun <T, R> unifySignatures(
+//    f0: Feature0<T, T>,
+//    f1: Feature1<T, Expect<R>.() -> Unit, T>
+//): List<Triple<String, Expect<T>.(Expect<R>.() -> Unit) -> Expect<T>, Boolean>> {
+//    val f0WithSubAssertion: Expect<T>.(Expect<R>.() -> Unit) -> Expect<T> =
+//        { f: Expect<R>.() -> Unit -> (f0.lambda)() }
+//    return listOf(
```

Comment:
*Edit* see comment to your question above

Reply:
Don't think it would fix it, my problem begin with the signature of the method itself, the inside I got it in a good state. (And I think we should actually -> Expect<T> since we don't use transform/others as state in other comment